lib/spack/spack/schema/ci.py

```diff
@@ -51,7 +52,7 @@
 
 submapping_schema = {
     "type": "object",
-    "additinoalProperties": False,
+    "additionalProperties": False,
```

Comment:
How odd, that nothing so far (unit tests, running pipelines) turned up this typo :thinking: